envoy.dependency.check/tests/test_integration_cves.py

```diff
@@ -64,8 +65,12 @@ def test_integration_cves(
          dict(new_callable=PropertyMock)),
         ("DependencyChecker.log",
          dict(new_callable=PropertyMock)),
+        ("DependencyChecker.loop",
+         dict(new_callable=PropertyMock)),
         ("DependencyChecker.session",
          dict(new_callable=PropertyMock)),
+        "DependencyChecker.install_reactor",
+        "DependencyChecker.setup_logging",
```

Comment:
this bit is slightly unrelated to the pr - other than the changes to aio.run.checker broke this (on current main)